tabular/src/autogluon/tabular/models/xt/xt_model.py

```diff
@@ -7,6 +7,10 @@ class XTModel(RFModel):
     """
     Extra Trees model (scikit-learn): https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html#sklearn.ensemble.ExtraTreesClassifier
     """
+    ag_key = "XT"
+    ag_name = "ExtraTrees"
+    ag_priority = 60
+    ag_priority_by_problem_type = {}
 
```

Comment:
Is this a typo?

Reply:
Not a typo, to avoid it inheriting RandomForest's ag_priority_by_problem_type, but since the RF value as redundant and I removed it, I will also remove this.